decidim-core/app/packs/stylesheets/decidim/_header.scss

```diff
@@ -46,15 +46,15 @@ header {
     }
 
     &__logo {
-      @apply col-span-3 xl:col-span-2 prose prose-img:max-h-16 prose-img:my-0;
+      @apply col-span-3 max-w-[11rem] xl:max-w-full xl:col-span-2 prose prose-img:max-h-16 prose-img:my-0;
     }
 
     &__logo-desktop {
-      @apply hidden lg:block;
+      @apply block;
     }
 
     &__logo-mobile {
-      @apply lg:hidden;
+      @apply hidden;
```

Comment:
Then, it's always hidden, it make no sense to have this class, and should be removed as well in the html view.